models/delete_models.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../db/connection');
+
+exports.removeComment = (comment_id) => {
+    return db.query(`
+    DELETE FROM comments
+    WHERE comment_id = $1
+    RETURNING *
```

Comment:
you are RETURNING whatever has been modifed from the database. You aren't testing it though. What is the purpose of returning in this case and why would we want to do this?

Reply:
I'm returning the deleted comment so I can check to see if it existed (in my controller, lines 6-8). This is so I can handle valid but non-existent comment ids. Not sure if this is best practice mind, or if I could do it without the psql return